scalyr_agent/configuration.py

```diff
@@ -250,6 +251,19 @@ def parse(self):
 
             self.__monitor_configs = list(self.__config.get_json_array("monitors"))
 
+            platform_controller = PlatformController.existing_platform()
+
+            # We add in special marker for monitor configuration objects which refer to built-in
+            # monitors so we can handle those differently inside the MonitorManager
```

Comment:
Let's resolve if we have to handle the implicit monitors specially.  We may not need this and other changes you made.